INSTALL.md

```diff
@@ -166,6 +166,16 @@ $ cd <competition>/etl
 $ ./deploy ~/data
 ```
 
+Note, running `deploy` with no arguments will give you the list of args available
+or required.  In some projects, more than just the `data` directory is needed.
+
+For most scripts, both `-c` and `-p` options are available.  The former uploads
+the spreadsheet only, not uploading attachments or creating wiki pages.  For the
```

Comment:
Just say "The latter allows...", not "For the latter, allows..."  (I.e., might as well match the construction you use for "The former...".)